ee/vault_ee.go

```diff
@@ -1,7 +1,8 @@
+//go:build !oss
```

Comment:
Is this right? not sure if this is part of the regex sed

Reply:
^ +1